packages/eslint-config-react-app/index.js

```diff
@@ -7,6 +7,8 @@
 
 'use strict';
 
+require('./patch');
```

Comment:
This file extends base too, so that should be enough, right?

In that case, we could maybe inline the `patch` file in `base` instead?

Reply:
I've moved logic to base instead